language_sentiment_text.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google LLC
```

Comment:
Could we open this PR against the language repo?  
https://github.com/googleapis/nodejs-language/

This is where the samples would live, so I'd love to see it opened there so we can discuss it in context.  Thanks!

Reply:
I'm on it!

I'll leave this here momentarily and then link of to Language and Speech.

Thanks so much @bcoe for already providing comments!

Reply:
Natural Language PR: https://github.com/googleapis/nodejs-language/pull/314

Simply generated samples and sent PR. Not mergeable because the tests aren't hooked into CI.

Speech PR coming right up!

Reply:
Speech PR!

https://github.com/googleapis/nodejs-speech/pull/453

I'll close this one, but let me collect @bcoe's feedback shortly first, e.g. if there are issues we need filed from the feedback